CEDS/PostgreSQL/0006-View-DimSeasDim-Create.sql

```diff
@@ -0,0 +1,112 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+DROP VIEW IF EXISTS xref.ceds_SeasDim;
+
+CREATE VIEW xref.ceds_SeasDim
+AS
+	WITH StateOrgEducationAddress AS (
+        SELECT
+			EducationOrganizationAddress.EducationOrganizationId,
+            EducationOrganizationAddress.City,
+			EducationOrganizationAddress.PostalCode,
+			StateAbbreviationDesc.CodeValue AS StateAbbreviation,
+			EducationOrganizationAddress.StreetNumberName,
+			EducationOrganizationAddress.ApartmentRoomSuiteNumber,
+			--
+			DescriptorConstant.ConstantName
+        FROM
+            edfi.EducationOrganizationAddress
+		INNER JOIN
+            edfi.Descriptor AS AddressType
+				ON EducationOrganizationAddress.AddressTypeDescriptorId = AddressType.DescriptorId
+		INNER JOIN
+            analytics_config.DescriptorMap
+				ON AddressType.DescriptorId = DescriptorMap.DescriptorId
+        INNER JOIN
+            analytics_config.DescriptorConstant
+				ON DescriptorConstant.DescriptorConstantId = DescriptorMap.DescriptorConstantId
+		LEFT JOIN
+			edfi.StateAbbreviationDescriptor
+				ON EducationOrganizationAddress.StateAbbreviationDescriptorId = StateAbbreviationDescriptor.StateAbbreviationDescriptorId
+		LEFT JOIN
+			edfi.Descriptor StateAbbreviationDesc
+				ON StateAbbreviationDescriptor.StateAbbreviationDescriptorId = StateAbbreviationDesc.DescriptorId
+    )
+	SELECT
+		CONCAT(
+			EducationOrganization.EducationOrganizationId, 
+			'-', EducationOrganizationAddress.AddressTypeDescriptorId,
+			'-', EducationOrganizationAddress.City,
+			'-', EducationOrganizationAddress.PostalCode,
+			'-', EducationOrganizationAddress.StateAbbreviationDescriptorId,
+			'-', EducationOrganizationAddress.StreetNumberName
+		) AS SeasDimKey,
+		EducationOrganization.NameOfInstitution AS SeaOrganizationName,
+		StateEducationAgency.StateEducationAgencyId AS SeaIdentifierSea,
+		CAST(NULL AS VARCHAR) AS StateAnciCode,
```

Comment:
Should the name be StateAnsiCode? (We use AnsiCode for the rest of the columns)
`'' AS StateAnsiCode` instead of `CAST(NULL AS NVARCHAR) AS StateAnciCode,`